code/game/machinery/cryo.dm

```diff
@@ -637,6 +637,20 @@ var/global/list/cryo_health_indicator = list(	"full" = image("icon" = 'icons/obj
 	put_mob(usr)
 	return
 
+/obj/machinery/atmospherics/unary/cryo_cell/verb/remove_beaker()
+	set name = "Remove beaker"
+	set category = "Object"
+	set src in oview(1)
+	if(usr.incapacitated() || usr.lying || usr.locked_to || !(iscarbon(usr) || issilicon(usr))) //are you cuffed, dying, lying, stunned or other
```

Comment:
` !(iscarbon(usr) || issilicon(usr))` are you sure about this?

Reply:
I tested with a human (worked), a silicon (worked), a mouse (didn't) and a ghost (didn't).
I imagine we don't want mouses removing beakers, and stuff like that.